### PR TITLE
Fixes CSS for Homescreen and Search Results

### PR DESCRIPTION
**Pull request by sdellis**
WIP: fixes responsive buttons and forms

### CODE REVIEW

**Comment by sdellis** [comment]
Screenshot 2024-10-31 at 2 46 45 PM
Screenshot 2024-10-31 at 2 46 31 PM
Screenshot 2024-10-31 at 2 46 07 PM
Screenshot 2024-10-31 at 2 45 51 PM

File: lib/dpul_collections_web/live/search_live.ex [hunk 1]
```diff
@@ -133,8 +135,31 @@ defmodule DpulCollectionsWeb.SearchLive do
...
+        <img
+          class="h-[350px] w-[350px] md:h-[225px] md:w-[225px]"
+          src="https://picsum.photos/350/350/?random"
+        />
+        <img
+          class="h-[350px] w-[350px] md:h-[225px] md:w-[225px]"
+          src="https://picsum.photos/350/350/?random"
+        />
+      </div>
+      <h2 class="underline text-xl font-bold pt-4"><%= @item.title %></h2>
```
Comment (hackartisan) [inline lib/dpul_collections_web/live/search_live.ex:162]: Seeing this as an h2 makes me curious; I would expect h1 to be the name of the site and h2 to be the title of the page. which if we had that would make these h3. We don't have a page title in our design, but would it be good to have one hidden from screen for accessibility reasons?

**Review by hackartisan — CHANGES_REQUESTED** [review]
This looks really good, I just had one thought!

File: lib/dpul_collections_web/live/search_live.ex (hunk 1 above)
Comment (sdellis) [reply lib/dpul_collections_web/live/search_live.ex:162]: @hackartisan that's a good point. I don't think we need to use the <h1> on every page for the name of the site, but the title of the page is important. In this case, I will add an <h1 class="sr_only"> to the page. Each result can then stay as an <h2>.

**Comment by sdellis** [comment]
Added Page Count feature depicted in wireframes.

Screenshot 2024-10-31 at 3 58 59 PM
Screenshot 2024-10-31 at 3 58 17 PM
Screenshot 2024-10-31 at 3 57 33 PM

**Review by tpendragon — APPROVED** [review]

**Review by hackartisan — APPROVED** [review]
Yay!

**Pull request merged**